examples/fibonacci.rs

```diff
@@ -1,10 +1,18 @@
 use chiquito::{
-    ast::ToField,
-    backend::halo2::{chiquito2Halo2, ChiquitoHalo2},
+    dsl::{
+        circuit, // main function for constructing an AST circuit
+        cb::*, // functions for constraint building
+    },
+    ast::{
+        expr::*,
+    },
     compiler::{
-        cell_manager::SingleRowCellManager, step_selector::SimpleStepSelectorBuilder, Compiler,
+        cell_manager::SingleRowCellManager, // input for constructing the compiler
+        step_selector::SimpleStepSelectorBuilder, // input for constructing the compiler
+        Compiler, // compiles AST to IR
     },
-    dsl::circuit,
+    ir::Circuit, // IR object that the compiler compiles to
+    backend::halo2::{chiquito2Halo2, ChiquitoHalo2}, // compiles to Chiquito Halo2 backend, which can be integrated into Halo2 circuit
 };
```

Comment:
In the future PR, we can define a "prelude" module for a quick start experience. 
Instead of importing all the required modules, users can get started with just one line, `use chiquito::prelude::*;`.

See this plotters example.
https://docs.rs/plotters/latest/plotters/prelude/struct.EmptyElement.html#example

Reply:
made this a new issue